src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -176,6 +176,7 @@
   <ItemGroup Condition="'$(TargetGroup)' == ''">
     <ProjectReference Include="..\..\System.Private.Uri\src\System.Private.Uri.csproj" />
     <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj" />
+    <ProjectReference Include="..\..\System.Buffers\src\System.Buffers.csproj" />
```

Comment:
At first thought I would be hesitate to introduce this dependency because System.Runtime.Extensions is a partial facade on various platforms and might cause some complications. Do we know if the gain here is worth possible dependency complications?

@KrzysztofCwalina what is your thoughts about this dependency? It would essentially pull it down int StdLib.